src/app/search/search-filters.ts

```diff
@@ -875,7 +876,20 @@ export function searchFilters(
           "BUCKET_SHIP",
           "BUCKET_HORN"
         ];
-        return item.bucket && cosmeticBuckets.includes(item.bucket.id.toString());
+        const CosmeticBucketTypes = [
+          "Shaders",
+          "Modifications",
+          "Emotes",
+          "Emblems",
+          "Vehicle",
+          "Ships",
+          "ClanBanners"
```

Comment:
Using the DIM bucket types, we should be able to create a single set for both D1 and D2. If we make it a Set (declared outside the function) we can also use `has` for the check.